.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,43 @@
+name: Run tests
+on: push
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgis/postgis:9.6-2.5
```

Comment:
If possible, later, we should tests with a matrix.

However, Postgresql10 (9.6 is not supported by PostgreSQL) and PostGIS 3.0 should be mandatory now.

Reply:
Since april 2021 we have some Execute statement in qwat-datamodel code what implies that 9.6 is now fully unsupported on newer versions of QWAT min postgres version is now 10, no special requirements on postgis side to my knowledge.

https://github.com/qwat/qwat-data-model/search?q=execute

https://www.postgresql.org/docs/current/sql-syntax-calling-funcs.html

Reply:
If the CI works correctly, this should fail :)

Reply:
@ponceta Are you sure it should fail ? `EXECUTE` existed on 9.6, aren't you talking about `EXECUTE FUNCTION` (which I think only comes in pg11, and is addressed in followup pr see https://github.com/qwat/qwat-data-model/pull/365/commits/ed4fcd3f2e3d1e66da543fe78074cadf5523385d)

However I agree we should bump to more up to date postgres, but would suggest to do that separately (so that this really is about reenable tests we used to run on travis as is)